### PR TITLE
Some datafixer methods and fields

### PR DESCRIPTION
**Pull request by YanisBft**
This really is uninteresting stuff

### CODE REVIEW

File: mappings/net/minecraft/util/JsonHelper.mapping [hunk 1]
```diff
@@ -131,7 +131,7 @@ CLASS net/minecraft/class_3518 net/minecraft/util/JsonHelper
 		ARG 1 name
 	METHOD method_15294 hasElement (Lcom/google/gson/JsonObject;Ljava/lang/String;)Z
 		ARG 0 object
-		ARG 1 lement
+		ARG 1 element
```
Comment (liach) [inline mappings/net/minecraft/util/JsonHelper.mapping:134]: :heart:

File: mappings/net/minecraft/datafixer/schema/Schema100.mapping [hunk 2]
```diff
@@ -2,7 +2,8 @@ CLASS net/minecraft/class_1222 net/minecraft/datafixer/schema/Schema100
 	METHOD <init> (ILcom/mojang/datafixers/schemas/Schema;)V
 		ARG 1 versionKey
 		ARG 2 parent
-	METHOD method_5195 targetEntity (Lcom/mojang/datafixers/schemas/Schema;Ljava/util/Map;Ljava/lang/String;)V
+	METHOD method_5195 targetEntityItems (Lcom/mojang/datafixers/schemas/Schema;Ljava/util/Map;Ljava/lang/String;)V
+		ARG 2 entityId
 	METHOD method_5196 targetItems (Lcom/mojang/datafixers/schemas/Schema;)Lcom/mojang/datafixers/types/templates/TypeTemplate;
```
Comment (liach) [inline mappings/net/minecraft/datafixer/schema/Schema100.mapping:7]: Hmm, we might consider different naming style for those that return type templates (used within supplier) and those that call schema register. I would keep `targetXxx` for the schema register one and rename these ones possibly? How do you think?

Comment (YanisBft) [reply mappings/net/minecraft/datafixer/schema/Schema100.mapping:7]: Tbh, I followed this specific method for naming all the others. How should the ones returning type templates be renamed?

Comment (liach) [reply mappings/net/minecraft/datafixer/schema/Schema100.mapping:7]: Hmm, you can just keep these names and we take care of these in a later refactor.

**Review by liach — APPROVED** [review]

**Review by Juuxel — APPROVED** [review]

**Pull request merged**